core/templates/dev/head/domain/statistics/PlaythroughImprovementCardObjectFactory.ts

```diff
@@ -87,8 +87,8 @@ angular.module('oppia').factory('PlaythroughImprovementCardObjectFactory', [
      * @returns {boolean} - Whether the improvement which this card suggests is
      * open, i.e., still relevant and actionable.
      */
-    PlaythroughImprovementCard.prototype.isOpen = function() {
-      return !this._isDiscarded;
+    PlaythroughImprovementCard.prototype.getStatus = function() {
+      return this._isDiscarded ? 'not_actionable' : 'open';
```

Comment:
Worth making these status strings into named constants and using them throughout?